draft-ietf-rats-architecture.md

```diff
@@ -170,16 +170,16 @@ Relying Party:
 
 Relying Party Owner:
 
-: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party
+: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party.
```

Comment:
```suggestion
: An entity (typically an administrator), that is authorized to configure Appraisal Policy for Attestation Results in a Relying Party
```

Reply:
Maybe there is a convention for not including period at the end of a sentence?

Reply:
RFC 4949 which we reference, uses periods.